StrangeCalculatorTask/src/main/java/Main.java

```diff
@@ -0,0 +1,13 @@
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        ConsoleReader consoleReader = new ConsoleReader();
+        Formula formula = new Formula();
+        do {
+            Calculator calculator = new Calculator();
+            System.out.println(calculator.calculate(consoleReader.readNext()));
+        }
+        while (consoleReader.hasNext() == true);
```

Comment:
No need to compare to true in condition, you could just leave it like:
while (consoleReader.hasNext());